haystack/nodes/file_classifier/file_type.py

```diff
@@ -62,6 +56,7 @@ def _estimate_extension(self, file_path: Path) -> str:
         :param file_path: the path to extract the extension from
         """
         try:
+            magic_import.check()
```

Comment:
Previously, we were just logging a debug message if `magic` could not have been imported, this will now probably raise an error - is this intended?

Reply:
Ah good point... I added it to the try-catch block but then I didn't catch the proper error :sweat_smile:  I'll fix line 62 to catch this one too!